src/m_refraction.cc

```diff
@@ -300,6 +300,48 @@ void refr_index_airMicrowavesGeneral(
   refr_index_air_group += n;
 }
 
+void refr_index_waterVisibleNIR(GriddedField3& complex_refr_index,
```

Comment:
Should probably be named complex_refr_indexWaterVisibleNIR

Add real value limit to documentation in methods.cc